buildSrc/build.gradle.kts

```diff
@@ -26,15 +26,23 @@ repositories {
     maven(url = "https://plugins.gradle.org/m2/")
 }
 
+spotless {
+    java {
+        target("src/**/*.java")
+        targetExclude("**/test/resources/**")
+        googleJavaFormat("1.22.0").reorderImports(false).skipJavadocFormatting()
```

Comment:
Bad, bad reviewer. I only noticed now that in the repo, and here, we have `reorderImports(false)`

do we want it like that? Does it do something unintended with the import order that we want to disable?

Reply:
@emilypgoogle

Reply:
I think the idea at the time was we wanted minimal formatting changing or the import ordering in this was incorrect? I don't actually remember but I'm pretty sure there was a reason.

Reply:
Changing it to `true` in the root `build.gradle` doesn't change the code, so maybe we are fine enabling it. 

@daymxn could you give it a try and check what's the output ?

Reply:
This didn't affect kotlin files anyways, but it seems our Java imports were already ordered correctly. We can enable it without any problems (as far as I can tell).